.gitignore

```diff
@@ -94,5 +94,5 @@ checkstyle-result.xml
 # unignore directories named titus-executor
 !*/titus-executor/
 
-test-standalone.log
```

Comment:
Why did you remove these two?

Reply:
the IDE told me to, since there's another rule for `*.log`

Reply:
neat.